recipes-extended/spi-minnowmax-board/files/modprobe.d/spi-minnow-at86rf230.conf.sample

```diff
@@ -0,0 +1,16 @@
+# These modules are required to enable AT86RF230 802.15.4 radio in the
+# MinnowBoard MAX
+#
+# To enable loading, link as below:
+#
+# install: ln -s /lib/modprobe.d/spi-minnow-at86rf230.conf.sample /etc/modprobe.d/spi-minnow-at86rf230.conf
+# to-autoload: ln -s /usr/lib/modules-load.d/at86rf230.conf.sample /etc/modules-load.d/at86rf230.conf
```

Comment:
@barbieri I asked this and the reason is: modprobe .conf should match the module name, autoload can be anything that describes the functionality (as you can see the autoload is not just spi-minnowmax- but also other SPI modules that are needed)